### ISSUE TITLE
The created tags for videos cannot be deleted/edited as a user. 

### ISSUE THREAD

**Issue by tjjmach** [mediacms-io/mediacms#434]
**Describe the issue**
The created tags for videos cannot be deleted or edited as a user. 
Shouldn't the users also be able to adjust their tags and delete them if necessary. I can't find any way to do this, except as an admin (admin panel).

**Comment by mgogoulos** [comment]
You are correct, you can only edit/delete them through the admin panel. But there are good news, the information which user created a tag is stored! So there could be two enhancements here: 

1. Users can see tags they have created, and edit or delete them
2. Whether users can create tags, could be a global option (configured by admins)
